src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class Lottery {
```

Comment:
Classes must have access modifiers otherwise default will be used, which isn't a good practice. Consider adding an access modifier here, typically 'public' for a class that should be accessible from outside its package.